tutorials-v5/visualization/animation-demo.md

```diff
@@ -0,0 +1,214 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Animation demos
+
+For more information about QuTiP see [http://qutip.org](http://qutip.org)
+
+```python
+from qutip import (ket, basis, sigmaz, tensor, qeye, mesolve,
+                   complex_array_to_rgb, about)
+import qutip as qt
+import numpy as np
+import matplotlib.pyplot as plt
+```
+
+
+```python
+# if this does not work on your environment, use qt.make_html_video
+# to see animations.
+%matplotlib notebook
+```
+
+## Time evolution of an operator
```

Comment:
I would change the first example to ''time evolution of a qubit'', and the second to ''time evolution of a cavity mode''